app/controllers/internal/syslog_drain_urls_controller.rb

```diff
@@ -46,6 +46,70 @@ def list
       [HTTP::OK, MultiJson.dump({ results: drain_urls, next_id: next_page_token }, pretty: true)]
     end
 
+    get '/internal/v5/syslog_drain_urls', :listv5
+
+    def listv5
+      prepare_aggregate_function
+
+      syslog_drain_urls = ServiceBinding.
+                          distinct.
+                          where(Sequel.lit('syslog_drain_url IS NOT NULL')).
+                          where(Sequel.lit("syslog_drain_url != ''")).
+                          select(:syslog_drain_url).
+                          order(:syslog_drain_url).
+                          limit(batch_size).
+                          offset(last_id).
+                          map(:syslog_drain_url)
+
+      bindings = ServiceBinding.
+                 join(AppModel.table_name, guid: :app_guid).
```

Comment:
I would stick to symbols when specifying tables, columns etc. so instead of `AppModel.table_name` you can use `:apps`.

Reply:
👍